web/cobrands/hampshire/assets.js

```diff
@@ -0,0 +1,75 @@
+(function(){
+
+if (!fixmystreet.maps) {
+    return;
+}
+
+var wfs_host = fixmystreet.staging ? 'tilma.staging.mysociety.org' : 'tilma.mysociety.org';
+var tilma_url = "https://" + wfs_host + "/mapserver/hampshire";
+
+var defaults = {
+    http_options: {
+        url: tilma_url,
+        params: {
+            SERVICE: "WFS",
+            VERSION: "1.1.0",
+            REQUEST: "GetFeature",
+            SRSNAME: "urn:ogc:def:crs:EPSG::27700"
+        }
+    },
+    geometryName: 'msGeometry',
+    srsName: "EPSG:27700",
+    body: "Hampshire County Council",
+    strategy_class: OpenLayers.Strategy.FixMyStreet
+};
+
+var highways_style = new OpenLayers.Style({
+    fill: false,
+    strokeColor: "#5555FF",
+    strokeOpacity: 0.1,
+    strokeWidth: 7
+});
+
+fixmystreet.assets.add(defaults, {
+    http_options: {
+        params: {
+            TYPENAME: "Road_Sections"
+        }
+    },
+    stylemap: new OpenLayers.StyleMap({
+         'default': highways_style
+    }),
+    usrn: {
+        attribute: 'SITE_CODE',
+        field: 'site_code'
+    },
+    road: true,
+    asset_item: 'road',
+    asset_type: 'road',
+    no_asset_msg_id: '#js-not-a-road',
+    always_visible: true,
+    non_interactive: true,
+    asset_group: "Roads/Highways",
+    actions: {
+        found: function(layer, asset) {
+            fixmystreet.message_controller.road_found(layer);
+        },
+        not_found: function(layer) {
+              fixmystreet.message_controller.road_not_found(layer);
+        }
+    }
+});
+
+fixmystreet.assets.add(defaults, {
+    http_options: {
+        params: {
+            TYPENAME: "salt_bins"
+        }
+    },
+    asset_type: 'spot',
+    asset_id_field: 'Asset_ID',
+    asset_category: ["Salt Bin"],
+    asset_item: 'salt bin'
+});
```

Comment:
Should this be setting any attribute with any data, or is it purely for location accuracy?

Reply:
Sorry if this wasn't clear - the `asset_id_field: 'Asset_ID'` was fine, that's for keeping the same thing selected if data is reloaded from the server; this was more asking if there should be more code to do anything when it's selected or not (not is fine, it was just a question).